docker-assets/manageiq.conf

```diff
@@ -0,0 +1,114 @@
+<Proxy balancer://evmcluster_ui/ lbmethod=bybusyness>
+  BalancerMember https://manageiq
+</Proxy>
+
+<Proxy balancer://evmcluster_websocket/ lbmethod=bybusyness>
+  BalancerMember ws://manageiq
+</Proxy>
+
+<Proxy balancer://evmcluster_ws/ lbmethod=bybusyness>
+  BalancerMember https://manageiq
+</Proxy>
+
+# Timeout: The number of seconds before receives and sends time out.
+Timeout 120
+
+# HTTP Start-up error log
+ErrorLog /var/log/httpd/miq_apache.log
+
+# Disable this section if using HTTP only
+RewriteEngine On
+Options SymLinksIfOwnerMatch
+RewriteCond %{SERVER_PORT} !^443$
+RewriteRule ^.*$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R]
+
+## ManageIQ SSL Virtual Host Context
+<VirtualHost *:443>
+  KeepAlive on
+
+  # To forward to the internal httpd servers in the app and ignore their self-signed certificates.
+  SSLProxyEngine On
+  SSLProxyVerify none
+  SSLProxyCheckPeerCN off
+  SSLProxyCheckPeerName off
+  SSLProxyCheckPeerExpire off
+
+  # The following redirects files must be included to handle most specific to least specific URLs.
+
+  ### Cockpit redirects
+  ProxyPreserveHost on
+  RequestHeader unset X-Forwarded-Proto
+  RequestHeader set X-Forwarded-Proto 'https' env=HTTPS
+  <LocationMatch "^/cws/cockpi(t[^/]+|t)?/socket$">
+    ProxyPassMatch "ws://localhost:9002/cws/cockpi$1/socket"
+  </LocationMatch>
+  ProxyPass /cws/ http://localhost:9002/cws/
+
+  ### API redirects
+  ProxyPass /api balancer://evmcluster_ws/api
+  ProxyPassReverse /api balancer://evmcluster_ws/api
+
+  ### UI redirects
+  RewriteCond %{REQUEST_URI} !^/ws
+  RewriteCond %{REQUEST_URI} !^/proxy_pages
+  RewriteCond %{REQUEST_URI} !^/saml2
+  RewriteCond %{REQUEST_URI} !^/api
+  RewriteCond %{REQUEST_URI} !^/ansibleapi
+  RewriteCond %{DOCUMENT_ROOT}/%{REQUEST_FILENAME} !-f
+  RewriteRule ^/ balancer://evmcluster_ui%{REQUEST_URI} [P,QSA,L]
+  ProxyPassReverse / balancer://evmcluster_ui/
+
+  ### WebSocket redirects
+  ProxyPass /ws balancer://evmcluster_websocket/ws
+  ProxyPassReverse /ws balancer://evmcluster_websocket/ws
+
+  ### Ansible redirects
+  # add a trailing slash to all requests to the ansible api
+  RewriteCond %{REQUEST_URI} ^/ansibleapi
+  RewriteCond %{REQUEST_URI} !(.*)/$
+  RewriteRule ^(.*)$ $1/ [R,NE,L]
+  ProxyPass /ansibleapi/ http://localhost:54321/api/
+  ProxyPassReverse /ansibleapi/ http://localhost:54321/api/
```

Comment:
This can all be removed. Because these configs are container specific we can take advantage of the fact that ansible will be a separate service.